tests/ts_simple_select_iso8601.erl

```diff
@@ -107,5 +109,17 @@ confirm() ->
               {ok, {_Cols, Data}} = ts_util:single_query(Conn, Qry),
 
               ?assertEqual(Tally, length(Data))
-      end, ?TESTS),
+      end, ?PASS_TESTS),
+
+    lists:foreach(
+      fun({{Op1, String1}, {Op2, String2}}) ->
+              Qry = lists:flatten(
+                      io_lib:format(QryFmt, [Op1, String1,
+                                             Op2, String2])),
+
+              RetMsg = ts_util:single_query(Conn, Qry),
+              %% Assert that RetMsg returns a tuple with error in first place {error, {}}
+              ?assertMatch({error, {_ErrCode, _ErrMsg}}, RetMsg)
```

Comment:
How about specifying the expected error code and message? Or at least the error code? You could match on it like the `Tally` in the success scenarios